io/serialport/25-serial.js

```diff
@@ -483,6 +500,24 @@ module.exports = function(RED) {
                 else {
                     done();
                 }
+            },
+            connect: function(port) {
+                if (connections[port]) {
+                    try {
+                        connections[port].connect();
+                    }
+                    catch(err) { }
```

Comment:
Do we really want to do nothing ? should this raise an error - or set node.status at least ?

Reply:
try catch was not necessory as we are using existing functions to connect so it should be handled in there.